lib/page.rb

```diff
@@ -77,6 +77,20 @@ def self.execute_script(script)
       Driver.execute_script_driver(script)
     end
 
+    def self.evaluate_script(script)
+      Driver.evaluate_script script
+    end
+
+    def self.wait_for_ajax
```

Comment:
> would this make more sense in the driver object?

To me, it makes sense to have it available in the Page object encapsulating the Driver.

IMHO, it reads better. I want to `wait_for_ajax` on the `Page` object.